server/routes/index.js

```diff
@@ -0,0 +1,22 @@
+import Businesses from '../controller/business';
```

Comment:
You have some lines of code on line 3, 5,7 that is not been used. You could remove them if they are not necessary here.

Reply:
Okay thank you samson. I will take that off now. I appreciate.